requirements/common.in

```diff
@@ -52,3 +52,9 @@ inflection==0.5.1
 
 # Change detection tooling
 mozdetect==0.0.9
+
+# Statistical methods for performance analysis
+scipy==1.15.3
+numpy==2.2.6
```

Comment:
Could we see if we can remove the numpy/scipy requirements? We already have numpy and scipy and a major version update to numpy could be problematic for other tooling.

If needed, we could also try downgrading the cliffs-delta/kdepy ones to match the versions of numpy we currently have.